include/aws/http/private/h2_connection.h

```diff
@@ -68,6 +68,14 @@ struct aws_h2_connection {
          * Waiting for WINDOW_UPDATE to set them free */
         struct aws_linked_list stalled_window_streams_list;
 
+        /* List using aws_h2_stream.node.
+         * Contains all streams that are open, but are only sending data when notified, rather than polling
+         * for it (e.g. event streams)
+         * Streams are moved to the outgoing_streams_list until they send pending data, then are moved back
+         * to this list to sleep until more data comes in
+         */
+        struct aws_linked_list evented_streams_list;
```

Comment:
Just unclear about what is this for, are we doing this to support event stream over h2? Or we also support passing more data to the stream overtime?

Reply:
Both, the idea is that the data is not available at request time, but is instead a long running connection with data available sparsely over time. Event streams, transcribe, etc.

Reply:
Great! Mike and I talked about it during h2 work planing, adding more data to stream overtime is one of the todos. Thanks for handling that!